docs/espace/tutorials/scaffoldCfx/scaffold.md

```diff
@@ -0,0 +1,131 @@
+---
+sidebar_position: 4
+title: Scaffold Conflux
+description: Learn how to leverage Scaffold-Eth 2 components to make your Conflux App
+keywords:
+  - Hardhat
+  - Smart Contracts
+  - Scaffold
+displayed_sidebar: eSpaceSidebar
+---
+
+[Scaffold Conflux](https://github.com/intrepidcanadian/conflux-scaffold) is an adaptation of Scaffold-ETH-2 (https://scaffoldeth.io/) whereby we have adjusted the template to allow you to deploy the contract on Conflux ESpace and leverage the components, integration of hardhat, and the quick deployment of Scaffold-ETH-2. In this tutorial, we will show you how to set-up Scaffold Conflux to better understand your smart contract and to leverage some of the beautiful react components available to quickly make your app on Conflux ESpace. 
+
+Watch this [video](https://youtu.be/33S0IjGGsQg) for full tutorial on how to create an app from scratch and deploy it live on Conflux ESpace.
+
+[![Watch Tutorial](https://img.youtube.com/vi/33S0IjGGsQg/0.jpg)](https://www.youtube.com/watch?v=33S0IjGGsQg)
+
+## Clone Scaffold Conflux
+
+In your visual studio terminal, run the following to clone the repo and install the dependencies.
+
+```bash
+git clone https://github.com/intrepidcanadian/conflux-scaffold
+yarn install
+```
+In the folder directory of `conflux-scaffold` you will have the following:
+
+![Folder Directory](../img/CfxScaffoldFolderStructure.png)
+
+The repository contains two main sections. The first section is for hardhat. 
+- You can adjust your smart contract solidity files in /conflux-scaffold/packages/hardhat/contracts 
+- For example, the current sample solidity file is "YourContract.sol" 
+- Start up your blockchain for testing purposes. We do this to easily create accounts and interact with the smart contract without deploying it on testnet (i.e. RPC limits) and Conflux Espace (i.e. gas fees)
+
+```bash
+yarn chain
+```
+
+Once the chain is set up, open up a new terminal. You can test your smart contract and then deploy it 
+
+```bash
+yarn hardhat:test
+yarn deploy
+```
+
+There are several files that are relevant when deploying. 
+
+- One is the deployment script in /conflux-scaffold/packages/hardhat/deploy
+- The other is the configuration of which networks you are deploying on. /conflux-scaffold/packages/hardhat/hardhat.config.ts
+- The default deployment network is hardhat but you can change it to Conflux Espace or Conflux Testnet by doing the following
+
+```bash
+yarn deploy --network ConfluxEspace
+```
+
+```bash
+yarn deploy --network confluxEspaceTest
```

Comment:
This network configuration is defined in hardhat. I suggest the network definition would be better to be consistent with viem definition

```suggestion
yarn deploy --network confluxESpaceTest
```